modules/ROOT/pages/_partials/concepts/sync-function.adoc

```diff
@@ -39,53 +39,70 @@ Any validation checks will probably fail because of the missing properties, so b
 
 :note-for-prototype: [This prototype shows the default Sync Function -- see <<lbl-args>> for more on the arguments.]
 
-== Sync Function Prototype
+[#ex-sync-function]
+== Sync Function Examples
 
-[#ex-prototype]
-.Prototype Sync Function
+Couchbase Sync Gateway defines a Sync Function at the `collection` level. 
+Defining at this level helps simplify data management and improve data reliability. 
+Each collection in the system allows for only one Sync Function, which enables the specification of Access Control rules.
+
+.Default Sync Function
 ====
 
-[{tabs}]
 =====
-
-Version 3.x::
-+
 --
 
 [source, javascript]
 ----
-function (doc, oldDoc, meta) { // <.>
-   channel(doc.channels); // <.>
+Sync (doc, oldDoc, meta) { 
```

Comment:
careful. This is Javascript: `function` was correct.
(Priya was handwaving in her comment on JIRA. The syntax of the sync functions hasn't changed, that we know of.)